nodejs/uws-binary/package.json

```diff
@@ -5,7 +5,11 @@
     "build": ""
   },
   "main": "uws.js",
-  "description": "Highly scalable WebSocket server library",
+  "description": "Highly scalable WebSocket server library - binaries for Linux/Darwin",
+  "engines": {
+    "node": "4.4.3 5.11.0 6.0.0"
```

Comment:
Same goes here, it is actually compatible with 4.x, 5.x, 6.x - only the major release numbers matter